### ISSUE TITLE
plot_network_graph() only plot target genes

### ISSUE THREAD

**Issue by zhenzuo2** [quadbio/Pando#56]
Hi,

The dimension of `seurat_object@grn@networks$glm_network@graphs$umap_graph` is the same as number of target genes saved in `seurat_object@grn@networks$glm_network`. 

As a result, only TFs that are also target genes were plotted. Is this an undocumented feature?

Thanks.

**Comment by joschif** [comment]
Yes, this is true for the umap layout, since here genes are represented by their regulators in the GRN. If you use another (e.g. forece-directed) layout for the GRN then Pando keeps all genes
